docs/certificates.md

```diff
@@ -0,0 +1,214 @@
+# Configuring certificates for Cloudformation AWS EC2 sidecars
+
+You can use Cyral's default [sidecar-created
+certificate](https://cyral.com/docs/sidecars/deployment/certificates#sidecar-created-certificate) or use a
+[custom certificate](https://cyral.com/docs/sidecars/deployment/certificates#custom-certificate) to secure
+the communications performed by the sidecar.
+
+In this page, we discuss the various certificate scenarios and their deployment instructions. Please refer to the [Sidecar certificate types](https://cyral.com/docs/sidecars/deployment/certificates#sidecar-certificate-types) documentation to understand the certificates used by the sidecar.
+
+## Deployment Scenarios
+
+There are several different deployment scenarios when it comes to certificates and the sidecar. Some are more suitable for a POC deployment, while others are geared more towards a production deployment.
+
+### Sidecar Created Certificates
+
+This method is more suitable for POC deployments as each sidecar instance will generate its own custom certificates.
+
+In this deployment scenario, each sidecar instance will generate its own self signed certificates during deployment. These certificates are NOT shared between instances in deployments where the sidecar is scaled to 2 or more instances.
```

Comment:
What about adding more info here adding some light to the fact that the sidecar will use this option automatically and the user don't need to change anything in the deployment template to use the sidecar-created certificate?